doc/man/sphinx-build.rst

```diff
@@ -89,6 +89,26 @@ Options
    See :ref:`builders` for a list of all builders shipped with Sphinx.
    Extensions can add their own builders.
 
+.. _make_mode:
+
+.. option:: -M buildername
```

Comment:
Sphinx only recognize `-M` option only if it is placed at first.
```
$ sphinx-build -v -M html . _build/html
Sphinx v1.7+
Usage: sphinx-build [options] sourcedir outdir [filenames...]

Filename arguments:
  without -a and without filenames, write new and changed files.
  with -a, write all files.
  with filenames, write these.

sphinx-build: error: no such option: -M
```

Reply:
This prevents doing things like this:

```
make SPHINXBUILD="python -msphinx -W" -C Doc/api/ html
```

in place of just:

```
make -C Doc/api/ html
```

in order to add extra arguments to ``sphinx-build`` (here ``-W`` for a strict run).

This worked on older versions of Sphinx as the ``Makefile`` did not use the ``-M`` argument.

Reply:
Solved my own issue, the new ``Makefile`` design lets me use ``SPHINXOPTS = -W`` which is a cleaner solution.

Reply:
@peterjc you can use ``O`` shortcut, ``make -C Doc/api/ html O="-W"``